src/BoundingBoxCameraSensor.cc

```diff
@@ -0,0 +1,525 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#include <mutex>
+#include <memory>
+
+#include <ignition/common/Console.hh>
+#include <ignition/common/Profiler.hh>
+#include <ignition/common/Image.hh>
+#include "ignition/sensors/RenderingEvents.hh"
+#include "ignition/sensors/SensorFactory.hh"
+#include "ignition/sensors/BoundingBoxCameraSensor.hh"
+#include <ignition/rendering/BoundingBoxCamera.hh>
+#include <ignition/transport/Node.hh>
+#include <ignition/transport/Publisher.hh>
+#include <ignition/msgs.hh>
```

Comment:
```suggestion
#include "ignition/sensors/BoundingBoxCameraSensor.hh"

#include <memory>
#include <mutex>

#include <ignition/common/Console.hh>
#include <ignition/common/Image.hh>
#include <ignition/common/Profiler.hh>
#include <ignition/msgs.hh>
#include <ignition/rendering/BoundingBoxCamera.hh>
#include <ignition/transport/Node.hh>
#include <ignition/transport/Publisher.hh>

#include "ignition/sensors/RenderingEvents.hh"
#include "ignition/sensors/SensorFactory.hh"
```

https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes